src/main/scala/rocks/muki/graphql/GraphQLCodegenPlugin.scala

```diff
@@ -22,6 +22,10 @@ object GraphQLCodegenPlugin extends AutoPlugin {
 
     val graphqlCodegenPackage =
       settingKey[String]("Package for the generated code")
+
+    val graphqlCodegenImports =
+      settingKey[Seq[String]]("Additional imports to add to the generated code")
```

Comment:
Typing this as a string is probably a good idea. scalameta is an implementation detail.
@jonas WDYT?